src/routes_public.ts

```diff
@@ -33,10 +34,16 @@ const routes = [
   },
 
   {
-    path: '/sign-in',
+    path: constants.publicPaths.SIGN_IN,
     name: 'SIGN IN',
     Component: SignInPage,
     isRhs: true,
   },
+  {
+    path: constants.publicPaths.RESET_PASSWORD,
+    name: '',
```

Comment:
`name` needs to be an empty string, so that the reset password route doesn't appear in the navbar.